libraries/abstractions/common_io/include/iot_uart.h

```diff
@@ -60,9 +60,10 @@
  */
 typedef enum
 {
-    eUartCompleted = IOT_UART_SUCCESS,            /*!< UART operation completed successfully. */
-    eUartLastWriteFailed = IOT_UART_WRITE_FAILED, /*!< UART driver returns error when performing write operation. */
-    eUartLastReadFailed = IOT_UART_READ_FAILED,   /*!< UART driver returns error when performing read operation. */
+    eUartWriteCompleted,        /*!< UART operation write completed successfully. */
+    eUartReadCompleted,         /*!< UART operation read completed successfully. */
```

Comment:
I like this change. But this problem happens on all the peripheral. We may want to discuss it first.